packages/react-ui-core/src/Map/Map.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import mapboxgl from 'mapbox-gl/dist/mapbox-gl'
+
+export default class Mapbox extends Component {
+  static propTypes = {
+    size: PropTypes.string,
+    color: PropTypes.string,
+    theme: PropTypes.object,
+    token: PropTypes.string,
+    center: PropTypes.array,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  componentDidMount() {
+    mapboxgl.accessToken = this.props.token
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'mapbox://styles/mapbox/streets-v9',
```

Comment:
we should not be hardcoding the map styles, these components should be flexible enough to be customized from the outside in.